package.json

```diff
@@ -35,5 +35,8 @@
   },
   "engines": {
     "node": ">=16.0.0"
+  },
+  "dependencies": {
+    "@emotion/css": "^11.10.6"
```

Comment:
Remove this dependency from the general package.json and install @emotion/css only in react project.

Reply:
Done.